docs/cli.md

```diff
@@ -636,6 +636,10 @@ This command is also available as a pre-commit hook. See [pre-commit hooks]({{<
 poetry check
 ```
 
+### Options
+
+* `--lock`: Verifies that `poetry.lock` exists for the current `pyproject.toml`.
```

Comment:
I see, I might have misunderstood the original suggestion.

I guess I've seen CI pipelines have a check step with something along the lines of `(1)`:

```
* poetry check
* poetry lock --check
```

In the current state of this PR, that can likely be simplified with just `(2)`:

```
* poetry check --lock
```

Changing it might lead to this scenario `(3)`:

```
* poetry check --lock
* poetry check
```

I'm leaning to scenario `(2)` from a UI design perspective, but I'm open to suggestions.

Reply:
@radoering FWIW, as the one who wrote #6756 I was hoping for `(2)` with the idea being that we simplify having to always run two commands to validate a Poetry environment fully, into one command. I think this makes the CI simpler and it's also simpler to teach.